lib/mediators.rb

```diff
@@ -6,4 +7,13 @@
 require_relative 'mediators/entities/mediator'
 require_relative 'mediators/api'
 
-OTR::ActiveRecord.configure_from_file! File.expand_path("../../config/database.yml", __FILE__)
+LOGGER = Logger.new(STDOUT)
+LOGGER.level = Logger::DEBUG
+
+if ENV['DATABASE_URL']
```

Comment:
Yes - there's a command you run from your local machine that sets env vars --- documentation is at https://tsuru-client.readthedocs.io/en/latest/reference.html#environment-variables